packages/driver/cypress/e2e/issues/1244.cy.js

```diff
@@ -80,5 +88,16 @@ describe('issue 1244', () => {
       cy.get('#dom').should('contain', 'DOM')
       cy.url().should('include', 'dom.html')
     })
+    */
+
+    it('does not strip link_parent', () => {
```

Comment:
I kept the commented out test, as it's the reproduction case for https://github.com/cypress-io/cypress/issues/28681

Reply:
@cacieprins Is there more than this line that chagnes? `cy.get('#dom').should('contain', 'DOM')` which we removed in a few of the other tests?

Reply:
The `_parent` test elements were changed in the fixture to load `empty.html` instead of `dom.html` - so they would not load the cached `dom.html` that does not have the cross-origin support script. The tests that used to assert that `dom.html` was loaded were changed to assert that `empty.html` was loaded instead.